tests/unit/tf/transformers/test_block.py

```diff
@@ -27,6 +28,8 @@ def test_import():
 
 @pytest.mark.parametrize("run_eagerly", [True])
 def test_retrieval_transformer(sequence_testing_data: Dataset, run_eagerly):
+    set_random_seed(42)
```

Comment:
we also have a fixture for setting a seed as an alternative mechanism https://github.com/NVIDIA-Merlin/models/blob/7f6170ca88848668ea5f54ca029e58f686163425/tests/unit/tf/_conftest.py#L45